src/promptflow-evals/promptflow/evals/synthetic/adversarial_simulator.py

```diff
@@ -115,7 +115,7 @@ async def __call__(
         api_call_retry_sleep_sec: int = 1,
         api_call_delay_sec: int = 0,
         concurrent_async_task: int = 3,
-        jailbreak: bool = False,
+        _jailbreak_type: Optional[str] = None,

```

Comment:
Was this released ? If yes will this break existing customers ?

Reply:
This is the change we discussed earlier in this PR and in the chat.

If a user is specifying the jailbreak type with this param in AdversarialSimulator directly, then yes, it will be breaking. But they should be using Jailbreak and the new XPIA simulators instead of calling AdversarialSimulator with those params directly.

Reply:
2 ways to handle it 👍 
- Introduce breaking change and make it explicit in CHAGELOG that next version will include breaking change.
- Make it backward compatible by looking for keyword arg `jailbreak` (since customer on older version will be passing it) and remapping it to _jailbreak_type and raising a warning that it will be deprecated in next version.

Reply:
I went with #1 since we want users to move away from using AdversarialSimulator directly to do jailbreak anyway